### ISSUE TITLE
is it possible to use wildcard

### ISSUE THREAD

**Issue by triple7coder** [kgretzky/evilginx2#1056]
#### REPORT ONLY BUGS OR FEATURE SUGGESTIONS.

Is there any possible way I can access my link with anything.domain.com?

**Comment by matejsmycka** [comment]
yes, in the DNS settings, they can redirect to your some.domain.com.

**Comment by ttinkywinky45** [comment]
**Hi 👽 ! If you are interested in ⚙️Advanced Phishing⚙️ and the ✨Evilginx3✨ tool then I offer you Google Gmail and Custom Phishlets with a ✅Free Tutorial✅ which arrives if the 📈Telegram reaches 50 Subscribers📈
⏳ GOOGLE GMAIL PHISHLET 2024 WITH PROOF ⏳
 ->
⛔https://www.youtube.com/watch?v=nA47UknIE6A⛔
---
🔓 Ask your questions and a free tutorial from 50 subs on telegram !!!
->
📩https://t.me/Noremorse1011📩**

![evilginx_background](https://github.com/kgretzky/evilginx2/assets/144027798/0e448f68-ecdc-49a4-a4f4-23dbed7cb971)

**Comment by Hosted-EU** [comment]
Join new Evilginx Group Chat 🔥https://t.me/evilginx_group
